client/lib/site/index.js

```diff
@@ -86,46 +87,7 @@ Site.prototype.set = function( attributes ) {
  * Update any computed attributes
  */
 Site.prototype.updateComputedAttributes = function() {
-	/**
```

Comment:
I think there are a handful of unused dependencies after removing this block of logic. I've not checked out the branch yet, but at a glance at least `trim` and `config`.